packages/edit-site/src/components/editor/index.js

```diff
@@ -125,25 +129,31 @@ function Editor() {
 	// and Query Pagination blocks.
 	const blockContext = useMemo(
 		() => ( {
-			...page.context,
-			query: page.context.query || { categoryIds: [] },
+			...page?.context,
```

Comment:
[TIL about the null propagation operator](https://ponyfoo.com/articles/null-propagation-operator) 

Very cool!